content/plugins/current/main/bridge.md

```diff
@@ -82,9 +66,44 @@ If the bridge is missing, the plugin will create one on first use and, if gatewa
 * `enabledad` (boolean, optional): enables duplicate address detection for the container side veth. Defaults to false.
 * `macspoofchk` (boolean, optional): Enables mac spoof check, limiting the traffic originating from the container to the mac address of the interface. Defaults to false.
 
-*Note:* The VLAN parameter configures the VLAN tag on the host end of the veth and also enables the vlan_filtering feature on the bridge interface.
+## VLAN capabilities
+
+The `vlan` and `vlanTrunk` parameters are mutually exclusive.
+
+The `vlan` parameter configures the VLAN tag on the host end of the veth and also enables the vlan_filtering feature on the bridge interface. The VLAN_DEFAULT_PVID of the bridge (when set), is added by default on the port as a VID untagged on egress. This is in most cases not desirable. Use the `preserveDefaultVlan` parameter to remove it.
```

Comment:
It would be nice to try and explain this with `VLAN_DEFAULT_PVID` as a side-note, just because it is not assured that a reader will understand what it means.
E.g. it is not clear who sets it in the first place.
But, as commented in the next comment, I would suggest to avoid adding bridge details here, just explain the configuration abstracted in this plugin and its outcomes.

Also, could you please clarify what "default"  `preserveDefaultVlan` refers to?
And what is its default value (when it does not appear in the config)?

Maybe it is worth trying to explain what are the side effects of having this set or not.